### DISCUSSION TITLE
how to sum the customized field in inventory

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#17082] [How to]
<img width="1328" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/18006681/c69f684a-ab5c-40ba-aa76-94e69c31f741">
anyone knows how to sum the customized field in inventory? please help advise

**Comment by vovpff** [comment]
This could be done by js coding. You can't do it from gui

**Reply by newminder** [reply]
I fixed the calculation, but do not know how to sum... I tried similar codes, but does not work, could you help tell me how?
<img width="582" alt="image" src="https://github.com/YetiForceCompany/YetiForceCRM/assets/18006681/3197f4ae-f35e-4008-87e8-bd2151b68775">
